CrowsZA/functions/fn_scatterPatternLine.sqf

```diff
@@ -0,0 +1,74 @@
+/*/////////////////////////////////////////////////
+Author: Crowdedlight+WindWalker
+			   
+File: fn_scatterPatternLine.sqf
+Parameters: center pos in AGL, number of spots to generate, offset in meters between points, altitude
+Return: array of positions for TP each player
+
+Generates an list of 3d positions for an line pattern. Used together with scatterTeleport
+
+*///////////////////////////////////////////////
+params ["_targetPos", "_amount", "_offset", "_targetAltitude", "_direction"];
+
+// create array of _amount positions to fill with teleport positions, We don't use the first position which is the targetPos, but build around it. 
+private _tpPositions = [];
+
+// line with distance of _playerOffset
+// direction vector in meters
+private _di = 0;
+private _dj = 0;
+
+//i is x ,j is y
+private _dirConstant = 0;
+
+switch (_direction) do {
+	case "North": { _dirConstant = 1};
```

Comment:
Dialog selector supports "name" and "pretty_name". So we can do "north-east" or another variant without space and capitalization while using "pretty_name" to show to the user, like: "North East"

Reply:
Example: https://github.com/Crowdedlight/Crows-Zeus-Additions/blob/main/CrowsZA/functions/fn_aceDamageToUnit.sqf#L92